pkg/api/api.go

```diff
@@ -52,6 +56,15 @@ func NewScratchDataAPI(
 		return nil, err
 	}
 
+	apiKeyTTL := conf.API.APIKeyCacheTTL
+	if apiKeyTTL == 0 {
```

Comment:
Does a TTL of 0 mean "we do not cache anything"? If so, then i'm okay removing this if-statement so that there's some way to disable the cache.

Reply:
For this library, the answer seems to be yes and no? While the item will instantly expire, any lookup that is less than an hour https://github.com/jellydator/ttlcache/blob/v3/cache.go#L537 will succeed, but return an item that is expired, which you could check with `item.ExpiresAt() < time.Now()`.